app/jobs/reporting/run_hud_report_job.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module Reporting
+  class RunHudReportJob < BaseJob
+    def perform(class_name, questions, report_id)
+      report = HudReports::ReportInstance.find(report_id)
+      report.update(state: 'Started', started_at: Time.current)
```

Comment:
Is it worth bringing in AASM for this?  It might make it cleaner. https://github.com/aasm/aasm